main/streams/streams.c

```diff
@@ -1418,8 +1418,11 @@ PHPAPI zend_string *_php_stream_copy_to_mem(php_stream *src, size_t maxlen, int
 			ptr += ret;
 		}
 		if (len) {
-			*ptr = '\0';
-			ZSTR_LEN(result) = len;
+			ZSTR_VAL(result)[len] = '\0';
+			//Only truncate if the savings are large enough
```

Comment:
I believe that we still do not use C++ style comments, so please change to C style comments (`/* … */`).

Reply:
Fixed